docs/troubleshooting/go-generate-rbac-errors.md

```
@@ -0,0 +1,34 @@
+# go generate RBAC Resource Errors
+
+## Symptom
+
+`go generate ./...` fails with panic:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)